cmd/keytransparency-monitor/main.go

```diff
@@ -66,10 +69,19 @@ func main() {
 	}
 	ktClient := pb.NewKeyTransparencyClient(cc)
 
-	config, err := ktClient.GetDirectory(ctx, &pb.GetDirectoryRequest{DirectoryId: *directoryID})
-	if err != nil {
-		glog.Exitf("Could not read directory info %v:", err)
+	// TODO(gbelvin): implement backoff for this
```

Comment:
Can you be more specific on what more needs to be implemented?

Reply:
Implemented and removed comment.